test/sol5/kyberStaking.js

```diff
@@ -2349,12 +2347,32 @@ contract('KyberStaking', function(accounts) {
             await stakingContract.withdraw(mulPrecision(100), {from: victor});
             await Helper.increaseBlockNumberBySendingEther(accounts[0], accounts[0], epochPeriod);
 
-            try {
-                await stakingContract.withdraw(mulPrecision(100), {from: victor});
-                assert(false, "throw was expected in line above.")
-            } catch (e) {
-                assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
-            }
+            await expectRevert.unspecified(
+                stakingContract.withdraw(mulPrecision(100), {from: victor})
```

Comment:
missing await.
and check revert message.

Reply:
This is reverted as mock DAO doesn't implement handleWithdrawal function, so there is no expected message here (actually messages are different for running test and coverage)